python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/enable_password.schema.yml

```diff
@@ -9,6 +9,10 @@ keys:
   enable_password:
     type: dict
     keys:
+      disabled:
+        type: bool
```

Comment:
```suggestion
        type: bool
        required: true
```

Reply:
This is not required. If someone sets hash_algorithm. This is different than the places where we have `enabled` to trigger a feature to be configured at all.